### PR TITLE
[TH2-1605] Working with estore

### PR DESCRIPTION
**Pull request by yarchiksmith**

### CODE REVIEW

File: src/main/java/com/exactpro/th2/act/events/EventStoreHandler.java [hunk 1]
```diff
@@ -185,7 +157,7 @@ private EventID storeEvent(EventDetails eventDetails)
 	{
 		Event event = createEvent(eventDetails);
 		try {
-			eventBatchRouter.send(EventBatch.newBuilder().addEvents(event).build(), "publish", "event");
+			eventBatchRouter.send(EventBatch.newBuilder().addEvents(event).build());
```
Comment (std000) [inline src/main/java/com/exactpro/th2/act/events/EventStoreHandler.java:160]: why did you delete sender attributes?

Comment (yarchiksmith) [reply src/main/java/com/exactpro/th2/act/events/EventStoreHandler.java:160]: Discussed

File: build.gradle [hunk 2]
```diff
@@ -51,7 +51,7 @@ dependencies {
 
     implementation 'com.exactpro.th2:common:3.13.4'
     implementation 'com.exactpro.th2:grpc-common:3.1.2'
-    implementation "com.exactpro.th2:grpc-hand:2.7.8"
+    implementation "com.exactpro.th2:grpc-hand:2.7.8-TH2-1605-966987667-SNAPSHOT"
```
Comment (std000) [inline build.gradle:54]: Add release version

**Review by std000 — APPROVED** [review]

**Review by std000 — APPROVED** [review]
